lib/PostMessageTransport.js

```diff
@@ -135,10 +135,12 @@ define(function (require, exports, module) {
 
     /**
      * Returns the script that should be injected into the browser to handle the other end of the transport.
+     * Includes a base tag to handle external protocol-less, linked files.
      * @return {string}
      */
     function getRemoteScript() {
-        return "<script>\n" +
+        return "<base href='https://mozillathimblelivepreview.net/bramble/dist'>\n" +
```

Comment:
The big question here is, how do we make this configurable?  @sedge, do you have any thoughts?  How can we get Thimble to pass use the right URL when we run, and if we don't get one, skip this step.